web/controllers/v1/bundles_controller.ex

```diff
@@ -139,7 +139,7 @@ defmodule Cog.V1.BundlesController do
   end
 
   defp error(:no_config) do
-    {:bad_request, %{error: "'bundle_config' not specified. Make sure to use a multipart form and to send your file in the 'bundle_config' field."}}
+    {:bad_request, %{error: "Invalid config."}}
```

Comment:
I think `"Missing bundle config."` would be a clearer error message given this function head is matching on `:no_config`.